### PR TITLE
✨ Use font-awesome 5

### LINKED ISSUE #453: ⬆️ Font Awesome Updaten
#### Describe your issue

Font Awesome is not up to date.
For that reason, some icons don't look like they should:

| BPMN-Studio | fontawesome.com |
| ----------- | --------------- |
| plug | plug |
| lock | lock |

#### Possible solution

Update Font Awesome.

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

### PR DESCRIPTION
**Pull request by alexanderkasten**
## What did you change?

This PR uses the @fortawesome/font-awesome package (font-awesome v5).
Most of used bootstrap icons are replaced with font-awesome icons.

Closes #453 
## How can others test the changes?

- checkout the branch
- `rm -rf node_modules`
- `npm i`
- `au run`

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [ ] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: src/modules/property-panel/indextabs/forms/sections/basics/basics.html [hunk 1]
```diff
@@ -86,7 +86,7 @@
     <div class="panel__heading">
       <span>Values</span>
       <button class="add-context-button" click.delegate="addEnumValue()">
-        <span class="fa fa-plus"></span>
+        <span class="fas fa-plus"></span>
```
Comment (Paulomart) [inline src/modules/property-panel/indextabs/forms/sections/basics/basics.html]: Should we replace usages of span with i?

Comment (alexanderkasten) [reply src/modules/property-panel/indextabs/forms/sections/basics/basics.html]: yes

**Comment by alexanderkasten** [comment]
@moellenbeck https://fontawesome.com/pro für schönere Icons.

**Comment by robpal94** [comment]
The _open lock_ looks a bit ambigous imo (and yeah, while I was typing this the PR gets merged but that wont stop me to finish this comment >:D ). Its a bit hard to tell, if the lock beneath the connected address on the left side is open or closed, since the icon is so small 

What  about this Icon instead? https://fontawesome.com/icons/lock-open?style=solid

**Pull request merged**